content/millennium/r4/documents/document-reference.md

```diff
@@ -98,13 +107,22 @@ List an individual DocumentReference by its id:
 
 #### Request
 
-    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference/197292845
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference/197411797
 
 #### Response
 
 <%= headers status: 200 %>
 <%= json(:R4_DOCUMENT_REFERENCE) %>
 
+#### Patient Authorization Request
+
+    GET https://fhir-myrecord.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference/197411797
```

Comment:
This should be pointing to ehr
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference?patient=12742400

Reply:
addressed